src/components/Sale/SaleButtons.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-container class="d-flex flex-row justify-center align-center ga-1">
+    <v-btn
+      size="small"
+      color="green lighten-1"
+      @click="$emit('open-dialog', 'calendar')"
+      :disabled="!props.sellObject.sellerName.id || !props.sellObject.buyerName.id"
+      >Calendar</v-btn
+    >
+
+    <v-btn
+      size="small"
+      color="primary"
+      @click="$emit('open-dialog', 'products')"
+      :disabled="!props.sellObject.date"
+    >
+      Products
+    </v-btn>
+
+    <v-btn
+      size="small"
+      color="#009688"
```

Comment:
Are we using this color anywhere else? I suggest extracting it in a constant somewhere if we are using it more than once.

Reply:
done